### ISSUE TITLE
add presets for common PL setting combinations, e.g. "allow only reactions"

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-meta#308]
I suggested in vector-im/element-web#20450:

> it might be useful to have some presets like "announcement room, users can only react" which sets message = mod, reaction = redact = default

saibotk commented:

> While vector-im/element-web#7492 added a settings switch, this still leads to weird UX, where Users can send reactions but are not allowed to remove them. (https://github.com/vector-im/element-web/issues/20450#event-5865321617)

**Comment by saibotk** [comment]
Wait, so is there an actual workaround using the devtools?
I did not figure out which power level i need to adjust to what value, so people can just remove their reactions.

**Comment by HarHarLinks** [comment]
Yes there is: `m.room.redaction` is needed and you're correct that this was missed in vector-im/element-web#7492. Set it to the same value as `m.reaction` in the room state.
